src/i18n/index.js

```diff
@@ -1,10 +1,10 @@
 import { enUS, ja, es, de, ptBR, fr, id } from "date-fns/locale";
```

Comment:
I think this is missing the pl import here.

Reply:
That's right! I fixed it.